src/main/java/io/github/jroy/happybot/events/star/StarMessages.java

```diff
@@ -26,23 +26,51 @@
 @SuppressWarnings("FieldCanBeLocal")
 public class StarMessages extends ListenerAdapter {
   private static final String NEW_GILDED_MESSAGE = "New Gilded Message";
-  private final String CREATE_TABLE = "CREATE TABLE IF NOT EXISTS starstats ( `id` INT(50) NOT NULL AUTO_INCREMENT , `userid` VARCHAR(50) NOT NULL , `stars` BIGINT(255) NOT NULL DEFAULT '0' , `gilds` BIGINT(255) NOT NULL DEFAULT '0' , `heels` BIGINT(255) NULL DEFAULT '0' , PRIMARY KEY (`id`)) ENGINE = InnoDB;";
+
+  private final String CREATE_STAT_TABLE = "CREATE TABLE IF NOT EXISTS starstats ( `id` INT(50) NOT NULL AUTO_INCREMENT , `userid` VARCHAR(50) NOT NULL , `stars` BIGINT(255) NOT NULL DEFAULT '0' , `gilds` BIGINT(255) NOT NULL DEFAULT '0' , `heels` BIGINT(255) NULL DEFAULT '0' , PRIMARY KEY (`id`)) ENGINE = InnoDB;";
```

Comment:
Can  we get some more descriptive names for these SQL commands? I can't make heads or tails of what kind of table this would be used for.

Reply:
Alternatively add comments to document the tables